src/TesApi.Web/Options/ApplicationInsightsOptions.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace TesApi.Web.Options
+{
+    /// <summary>
+    /// Default application insights configuration options
+    /// </summary>
+    public class ApplicationInsightsOptions
+    {
+        /// <summary>
+        /// Configuration section.
+        /// </summary>
+        public const string SectionName = "ApplicationInsights";
+
+        /// <summary>
+        /// Account name.
+        /// </summary>
+        public string AccountName { get; set; }
+        // TODO: add additional properties to run without a managed id?
```

Comment:
`ConnectionString`

https://learn.microsoft.com/en-us/azure/azure-monitor/app/asp-net-core?tabs=netcorenew%2Cnetcore6

Seems like if ConnectionString is specified, it should be used instead.  Thoughts?  Can you add this?

Reply:
Done